### PR TITLE
Day 2 Capstone M2: Basic app structure 

### PR DESCRIPTION
**Pull request by karlavdelgadof**
## Capstone M2: Basic App structure (HTML/CSS) :memo: :hammer_and_wrench:

These are the proposed changes :recycle:  for this project :arrow_down: :

- Created `header` and `footer`
- Created basic styling in `style.css` and set rules for the web app according to the provided wireframe :art: :heavy_check_mark: 
- Added favicon :100: 

### Note :warning: 
This task was distributed as follows:

- [Student A @karlavdelgadof](https://github.com/karlavdelgadof/Capstone-M2-JS/issues/17#issue-1317368020) :bust_in_silhouette: 
- [Student B @wayungi ](https://github.com/karlavdelgadof/Capstone-M2-JS/issues/16#issue-1317367725) :bust_in_silhouette:

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -3,9 +3,20 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
```
Comment (wayungi) [inline dist/index.html:6]: Please change the title to match the app name

File: src/index.html [hunk 2]
```diff
@@ -3,9 +3,25 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <link rel="shortcut icon" href="./src/favicon-32x32.png" type="image/x-icon" />
+    <title>Music App</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
```
Comment (wayungi) [inline src/index.html:9]: Please remove this test code

File: src/style.css [hunk 3]
```diff
@@ -1,7 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+Display:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap');
+
+* {
+  text-decoration: none;
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+
+  /* outline: 2px solid red; */
```
Comment (wayungi) [inline src/style.css:10]: Please remove comments from your code

File: src/style.css [hunk 4]
```diff
@@ -1,7 +1,84 @@
...
 body {
-  background-color: lightblue;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: rgba(238, 238, 238, 0.626);
+  background-blend-mode: soft-light;
+
+  /* background-image: url('./multimedia/bg.jpg'); */
```
Comment (wayungi) [inline src/style.css]: Please remove this comment

**Review by wayungi — CHANGES_REQUESTED** [review]
## Good work coming this far in the project development.
- Please find my recommendations for changes below

### Changes required
Please find the recommended changes below.

File: src/index.html (hunk 2 above)
Comment (karlavdelgadof) [reply src/index.html:9]: Done :heavy_check_mark:

File: dist/index.html (hunk 1 above)
Comment (karlavdelgadof) [reply dist/index.html:6]: Done :heavy_check_mark:

**Review by wayungi — APPROVED** [review]
Good work making the changes.

You are good to proceed.

**Pull request merged**